tests/_data/snapshots/license-evidence-dev_juice-shop.json.bin

```diff
@@ -5885,19 +5876,7 @@
           "comment": "as detected from PackageJson property \"homepage\""
         }
       ],
-      "evidence": {
-        "licenses": [
-          {
-            "license": {
-              "name": "file: LICENSE.MIT",
```

Comment:
this is a potential license evidence.

to be honest, i'd rather improve the pattern match in a later ticket/pr, 
than spending too much time with this initial base PR

Reply:
should be solved via  1dad58ecdcbe9a7ac39caf196b4b3bd7e534bed5